contracts/introspection/ERC165Query.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.4.20;
+
+
+/**
+ * @title ERC165Query
+ * @dev https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md
+ */
+library ERC165Query {
+  bytes4 constant INVALID_ID = 0xffffffff;
+  bytes4 constant ERC165_ID = 0x01ffc9a7;
```

Comment:
Consider adding a comment mentioning that this equals `bytes4(keccak256('supportsInterface(bytes4)')`.

Reply:
fixed